kubemarine/resources/configurations/compatibility/kubernetes_versions.yaml

```diff
@@ -46,6 +46,12 @@ compatibility_map:
     kubernetes-dashboard: v2.7.0
```

Comment:
We should probably remove v1.34.0, since we did not released it yet, we do not want to support it

Reply:
Removed v1.34.0 from supported versions